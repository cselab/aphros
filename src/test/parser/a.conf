
# asdf adsf 

set string dumplist p
set string dumplist $dumplist vx vy

include b.conf

#  adsfasdf 
set double e   3.1   


## adfa s

set string sub [sub]
set string sub_2 a${sub}b
set string sub_3 a$sub_2
set string sub_4 a${sub}_2b

set int subint 9
set double subdouble 9.1
set double d1 $subint
set double d2 ${subint}
set vect subvect 1 2 3
set string suball [$subint] [${subdouble}] [$subvect] ${subint}

set string suball2 ${subint}
