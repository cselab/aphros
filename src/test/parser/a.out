
Simple
set string a 1
set int b 1
set double c 1
set vect d 1 

TestFile
set string c a b   s c    
set string s 
set string sub [sub]
set string sub_2 a[sub]b
set string sub_3 aa[sub]b
set string sub_4 a[sub]_2b
set string suball [9] [9.1] [1 2 3 ] 9
set string suball2 9
set int b 123
set int subint 9
set double a 2.9
set double d1 9
set double d2 9
set double e 3.1
set double subdouble 9.1
set vect f 1 2 34 5 
set vect subvect 1 2 3 
